circle.yml

```diff
@@ -0,0 +1,3 @@
+machine:
+  ruby:
+    version: 2.3.5
```

Comment:
I think we should set this in the gemfile so that we can kill two birds with one stone. That way, all our environments will have the same ruby version

Reply:
Can I do this in another PR? It feels unrelated to this PR

Reply:
Works for me

Reply:
I don't think we need this file anymore since we're enforcing it in the `Gemfile`